packages/feedback-service/src/typedef.graphql

```diff
@@ -0,0 +1,52 @@
+type ProfileType {
+  kerberosID: String
+  name: String
+  email: String
+}
+
+type TimestampType {
+  createdAt: DateTime
+  createdBy: ProfileType
+  modifiedAt: DateTime
+  modifiedBy: ProfileType
+}
+
+type FeedbackType {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  spa: String
+  portalFeedback: Boolean
+  timestamp: TimestampType
+  title: String
+  feedbackType: String
+}
+
+input FeedbackInput {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  module: String
+  portalFeedback: Boolean
+  timestamp: TimestampInput
+  title: String
+  feedbackType: String
+}
+
+type Query {
+  # Query Definitions
+  listFeedback: [FeedbackType]
+  getFeedback(_id: String): FeedbackType
```

Comment:
Add one more query to getFeedback by any property. `getFeedbackBy`

Reply:
So you mean user can pass single property or can pass all the property  including _id ?

Reply:
You can pass a partial object of type `FeedbackInput`. And any properties that match the input object will be returned.